src/ngx_http_set_hashed_upstream.c

```diff
@@ -129,10 +241,67 @@ ngx_http_set_hashed_upstream(ngx_conf_t *cf, ngx_command_t *cmd, void *conf)
         return NGX_CONF_ERROR;
     }
 
+    hash_conf->ul = ul;
+    hash_conf->ul_var = 0;
+
+    if (ul->nelts <= 0) {
+        hash_conf->node_len = 0;
+        hash_conf->hash_nodes = NULL;
+
+    } else if (hash_type == set_misc_consistent_hash) {
+        hash_conf->node_len = ul->nelts * HASH_VNODES;
+        hash_nodes_size =
+            sizeof(ngx_http_set_hashed_upstream_consistent_hash_node) *
+            hash_conf->node_len;
+        hash_conf->hash_nodes = ngx_pcalloc(cf->pool, hash_nodes_size);
+
+        if (hash_conf->hash_nodes == NULL) {
+            return NGX_CONF_ERROR;
+        }
```

Comment:
@charliechang Sorry, my mistake. I thought the long line breaks were indentation problems.